### PR TITLE
h

### PR DESCRIPTION
**Pull request by IMS212**
h

### CODE REVIEW

File: fabric/build.gradle [hunk 1]
```diff
@@ -21,9 +21,9 @@ dependencies {
     modCompileOnly "mcp.mobius.waila:wthit-api:fabric-${rootProject.wthit_version}" // compile against the WTHIT API
     modRuntimeOnly "mcp.mobius.waila:wthit:fabric-${rootProject.wthit_version}" // run against the full jar
 
-    modRuntimeOnly "maven.modrinth:lazydfu:0.1.2"
-    modRuntimeOnly "maven.modrinth:suggestionproviderfabric:1.0.1"
-    modRuntimeOnly("com.terraformersmc:modmenu:3.2.1") {
+    modRuntimeOnly "maven.modrinth:lazydfu:0.1.3"
+    //TODO 1.19 modRuntimeOnly "maven.modrinth:suggestionproviderfabric:1.0.1"
```
Comment (wafflecoffee) [inline fabric/build.gradle:25]: https://modrinth.com/mod/isnrmn

Comment (IMS212) [reply fabric/build.gradle:25]: is this a direct successor?

Comment (wafflecoffee) [reply fabric/build.gradle:25]: dunno but it does the exact same thing

**Comment by Redd56** [comment]
h

**Comment by BeansBeefBroccoli** [comment]
h

File: fabric/build.gradle (hunk 1 above)
Comment (IMS212) [reply fabric/build.gradle]: Done

File: gradle.properties [hunk 2]
```diff
@@ -8,21 +8,21 @@ mod_version = 0.5.0
...
 
 # Api Versions
-fabric_api_version = 0.51.1+1.18.2
-scrt_version = 3.0.1
-scrt_version_forge = 3.0.1
-architectury_version = 4.2.50
-wthit_version = 4.8.0
+fabric_api_version = 0.57.0+1.19
+scrt_version = 4.0.0
+scrt_version_forge = 4.0.0
```
Comment (magneticflux-) [inline gradle.properties:17]:
```suggestion
scrt_version = 4.0.0-fabric
scrt_version_forge = 4.0.0-forge
```

**Review by magneticflux- — CHANGES_REQUESTED** [review]

**Comment by magneticflux-** [comment]
Bruh

Okay this patch actually fixes it:
```patch
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision ee8aa5b68d404a4564e997f77a4468396fdf80c7)
+++ b/gradle.properties	(date 1659742652922)
@@ -13,8 +13,8 @@
 
 # Api Versions
 fabric_api_version = 0.57.0+1.19
-scrt_version = 4.0.0-fabric
-scrt_version_forge = 4.0.0-forge
+scrt_version = 4.0.0
+scrt_version_forge = 4.0.0
 architectury_version = 5.7.28
 wthit_version = 5.6.1
 
Index: fabric/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fabric/build.gradle b/fabric/build.gradle
--- a/fabric/build.gradle	(revision ee8aa5b68d404a4564e997f77a4468396fdf80c7)
+++ b/fabric/build.gradle	(date 1659742652917)
@@ -16,7 +16,7 @@
 dependencies {
     modImplementation "net.fabricmc:fabric-loader:${rootProject.fabric_version}"
     modApi "net.fabricmc.fabric-api:fabric-api:${rootProject.fabric_api_version}"
-    modApi include("maven.modrinth:stonecutter_recipe_tags:${rootProject.scrt_version}")
+    modApi include("maven.modrinth:stonecutter_recipe_tags:${rootProject.scrt_version}-fabric")
     modApi "dev.architectury:architectury-fabric:${rootProject.architectury_version}"
     modCompileOnly "mcp.mobius.waila:wthit-api:fabric-${rootProject.wthit_version}" // compile against the WTHIT API
     modRuntimeOnly "mcp.mobius.waila:wthit:fabric-${rootProject.wthit_version}" // run against the full jar

```

**Pull request closed without merging**